learnrl/playground.py

```diff
@@ -195,8 +196,9 @@ def run(self,
                     self.env.render()
 
                 # Get playing agent (TurnEnv)
-                agent_id = self.env.turn(observation) if isinstance(self.env, TurnEnv) else 0
-                if agent_id >= len(previous):
+                turn_id = self.env.turn(observation) if isinstance(self.env, TurnEnv) else 0
+                agent_id = self.agents_order[turn_id]
+                if agent_id >= len(self.agents):
                     raise ValueError(f'Not enough agents to play environement {self.env}')
                 agent = self.agents[agent_id]
```

Comment:
Based on what Théo showed us, isn't it a good place to use the following code in order to avoid a check which should often be false?

```python
try:
  agent = self.agents[agent_id]
except IndexError:
  raise ValueError(f'Not enough agents to play environement {self.env}')
```

Reply:
Good idea !